keras_cv/layers/vit_patching.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class Patching(layers.Layer):
+    """
+    Layer to turn images into a sequence of patches for Vision Transformers from:
```

Comment:
It might be usefull to have standard layers for `image-to-patch` and  vice-versa! ex:

```python
class TransformImageToPatch
class TransformPatchToImage
```

cc. @LukeWood

Reply:
Quick question - why would we want to convert patches to images? Or did you mean patch merging like it's used in SwinT?

Reply:
Here is one cases in [maxim model](https://github.com/google-research/maxim/blob/main/maxim/models/maxim.py#L56-L72).

Reply:
If keras-team thinks so. However, later I realize that, those image-to-patch and patch-to-image are kind of maxim-design oriented. So, it isn't general. Apart from it, this ticket (https://github.com/tensorflow/tensorflow/issues/6847) should be reopened.